src/mwindow.c

```diff
@@ -20,7 +20,7 @@
 		: 32 * 1024 * 1024)
 
 #define DEFAULT_MAPPED_LIMIT \
-	((1024 * 1024) * (sizeof(void*) >= 8 ? 8192ULL : 256UL))
+	((1024 * 1024) * (sizeof(void*) >= 8 ? UINT64_C(8192) : 256UL))
```

Comment:
Not sure what's up with that UL, which *is* valid, but may not be what was intended.

Reply:
yeah, that `UL` looks very suspicious. maybe wrap it with `UINT64_C(...)` to avoid the implicit conversion in the conditional.